llvm/test/CodeGen/X86/vector-constrained-fp-intrinsics.ll

```diff
@@ -3011,25 +3011,26 @@ entry:
   ret <4 x double> %log2
 }
 
-define <1 x float> @constrained_vector_rint_v1f32() #0 {
+define <1 x float> @constrained_vector_rint_v1f32(ptr %a) #0 {
```

Comment:
change the name to constrained_vector_rint_v1f32_var (etc.) to match the naming convention for constant / non-constant tests